Jenkinsfile

```diff
@@ -228,11 +260,11 @@ pipeline {
           agent {
             docker {
               image "${nodeImage}"
+              reuseNode true
             }
           }
           steps {
             sh "rm -f storybook.zip"
-            sh 'make install'
```

Comment:
I can't see any reason why it should be necessary to install deps again at this point but happy to be corrected!